docs/vnext/openapi.yaml

```diff
@@ -460,26 +392,29 @@ components:
         target:
           type: string
           description: An absolute JSON Pointer (see [RFC 6901](https://datatracker.ietf.org/doc/html/rfc6901)) to the property causing the error.
-          example: /properties/foo
+          example: /type
     ResourceReference:
       type: object
       required:
         - referenceId

```

Comment:
If ReferenceId should be deleted then should identifiers be required here instead?